gateway/src/main/java/io/zeebe/gateway/grpc/ServerStreamObserver.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.gateway.grpc;
+
+import io.grpc.stub.StreamObserver;
+
+/**
+ * A simple extension of {@link StreamObserver}, meant to be used in conjunction with {@link
+ * io.grpc.stub.ServerCallStreamObserver}. In order to avoid depending on {@link
+ * io.grpc.stub.ServerCallStreamObserver}, which is experimental (as of now), we introduce this
+ * simple interface, and we can easily change the implementation whenever the experimental API is
+ * changed.
+ *
+ * @param <GrpcResponseT> the expected gRPC response type
+ */
+@SuppressWarnings("java:S119")
```

Comment:
It is a bit weird with the suppresswarnings to be honest

Reply:
Like I said, it's because sonar cloud is set up to warn you if you don't use all caps for your types. We can also change the SonarCloud config instead :+1: